<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QButtonGroupTestClass</class>
 <widget class="QDialog" name="QButtonGroupTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QButtonGroupTest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioButton_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
